src/installLogsCollector.js

```diff
@@ -108,6 +108,13 @@ function installLogsCollector(config = {}) {
 }
 
 function validateConfig(config) {
+  before(function () {
+    if (typeof config.printLogs === 'string'){
+      cy.log("printLogs configuration is no longer supported, consider using the new options within plugins: " +
```

Comment:
Message: 'cypress-terminal-report: WARN! printLogs configuration has been removed. Please check changelog in readme.'